meta-ostro/classes/ostro-image.bbclass

```diff
@@ -184,7 +184,8 @@ FEATURE_PACKAGES_ima = "packagegroup-ima-evm-utils"
 FEATURE_PACKAGES_iotivity = "packagegroup-iotivity"
 FEATURE_PACKAGES_devkit = "packagegroup-devkit \
     ${@ bb.utils.contains('IMAGE_FEATURES', 'java-jdk', 'mraa-java upm-java', '', d)} \
-    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa', '', d)} \
+    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa python-upm', '', d)} \
```

Comment:
Yes it was an issue in upm's cmake configurations that @pylbert fixed with
the 0.7.2 release. Commits probably should have been split for clarity.
On 5 Jul 2016 17:17, "Patrick Ohly" <notifications@github.com> wrote:

> In meta-ostro/classes/ostro-image.bbclass
> <https://github.com/ostroproject/meta-ostro/pull/225#discussion_r69592912>
> :
>
> > @@ -184,7 +184,8 @@ FEATURE_PACKAGES_ima = "packagegroup-ima-evm-utils"
> >  FEATURE_PACKAGES_iotivity = "packagegroup-iotivity"
> >  FEATURE_PACKAGES_devkit = "packagegroup-devkit \
> >      ${@ bb.utils.contains('IMAGE_FEATURES', 'java-jdk', 'mraa-java upm-java', '', d)} \
> > -    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa', '', d)} \
> > +    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa python-upm', '', d)} \
>
> That update was merged into ostro-os in parallel to me enabling the
> packages in the images - that explains why I didn't have it yet.
>
> The commit messages in meta-intel-iot-middlware only mention the version
> bump, so it wasn't obvious to me that something regarding the bindings
> might have changed.
>
> Has the root cause of the original problem with the Python bindings,
> unreliable builds, been identified and fixed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ostroproject/meta-ostro/pull/225/files/482d2d777d24bd6c885878873ed6ea51cd7412f2#r69592912>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfZZrSdXmObV9FHXrEbDnV2TK7ARgXFks5qSoOogaJpZM4JFLfQ>
> .
>